packages/shared/src/utils/file-utils.ts

```diff
@@ -480,6 +485,26 @@ export function replicateDir(
   return ops;
 }
 
+/** Copy a file from src to target, only replacing if unchanged */
+export function replicateFile(src: string, target: string) {
+  const srcExists = fs.pathExistsSync(src);
+  if (!srcExists) return Logger.error({ msg1: "File not found", msg2: src });
+  const srcStats = getFileStats(src);
+  // skip if target file same contents as src
+  if (fs.pathExistsSync(target)) {
+    const targetStats = getFileStats(target);
+    if (srcStats.md5Checksum === targetStats.md5Checksum) {
+      return;
+    }
+  }
+  // copy from src to target
+  fs.ensureDir(path.dirname(target));
```

Comment:
Thanks for the catch. As it happens this entire function is no longer used (had originally tested copying config file from deployment to app assets before realising it made more sense to write just the subset of config properties to file), but will keep for now in case we do move to separate build/runtime config files that can be copied over